fiftyone/utils/sam.py

```
@@ -647,270 +662,315 @@ class SegmentAnythingModel(fout.TorchSamplesMixin, fout.TorchImageModel):
+                "prompt_type": name of prompt type for the batch
+                "classes": a list of classes for each prompt
+        """
+        results = {}
+        for key in batch[0]:
+            results[key] = []
+
+        for item in batch:
+            for key, val in item.items():
+                if key == "point_labels":
+                    continue
+                elif key == "point_coords":
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `continue` statement

Remove unnecessary `elif`

(RET507)